openlibrary/core/ia.py

```diff
@@ -292,3 +298,90 @@ def add_subjects(self):
         subjects = [subject for c, subject in mapping.items() if c in collections]
         if subjects:
             self['subjects'] = subjects
+
+_ia_db = None
+def get_ia_db(configfile=None):
+    """Metadata API is slow.
+
+    Talk to archive.org database directly if it is specified in the
+    global configuration or if a configfile is provided.
+    """
+    if configfile:
+        from openlibrary.config import load_config
+        load_config(configfile)
+
+    if not config.get("ia_db"):
+        return
```

Comment:
Would it be more clear to return `None`? Or does convention suggest not specifying a return value?